### ISSUE TITLE
Some issue with the bulk tag deployment update tool

### ISSUE THREAD

**Issue by jsayers-bsc** [MotusDev/Motus-TO-DO#988]
Someone noticed that there are some detections on the tracks map in Kazakhstan that shouldn't be there, from the tagging location longitude being positive instead of negative. To quickly find and correct the non-negative values, I sorted the file and saved with the new sorting. After trying to upload, I have a couple suggestions. 

- Errors and warnings are reported using the row numbers of the _original_ file, not the modified file just uploaded. In the event that the new file has been sorted, this is quite confusing!

- > You have entered a start date in the past without providing a species. You may either enter a future start date or populate the species column. If you're not yet ready to provide this data you may omit this deployment from your upload template.

This error is displayed even if there is a species provided, but one that doesn't match the the species list. In this case, Oreothlypis ruficapilla was provided for Nashville warbler. If the species field is not blank, this error should instead read something like "You have entered a species name that is not supported". There is a warning to this effect (which is displayed in addition to the error) but only one of these is probably needed.

- > The markerType value does not exist. This information is required. 

Is this required any time band number / weight / etc are entered? The field description file doesn't indicate any constraints on this so presumably any non-empty field would dispel the warning?

**Comment by cjardine-bsc** [comment]
Re: Errors and warnings are reported using the row numbers of the original file, not the modified file just uploaded. In the event that the new file has been sorted, this is quite confusing!

Can you double check that? It shouldn't be possible that it's using the row numbers of the original file. The only way I can think of that happening is if there was a pervious upload that was incomplete and the second upload failed. But there should be a clear error if the upload fails.

**Comment by jsayers-bsc** [comment]
Here is the tag template downloaded from the site:

![image](https://user-images.githubusercontent.com/55001144/123674511-afc58a80-d80f-11eb-97ec-cd0b31a6df09.png)


Here is the re-ordered version I modified:

![image](https://user-images.githubusercontent.com/55001144/123674579-c66be180-d80f-11eb-8a6d-83c49fa9d96f.png)

And here is the list of error messages:

![image](https://user-images.githubusercontent.com/55001144/123674642-d5eb2a80-d80f-11eb-9ddd-c8125921c245.png)

**Comment by cjardine-bsc** [comment]
Can you send me the re-ordered file so I can test?

**Comment by cjardine-bsc** [comment]
I tested with the file you sent me and it worked correctly. 

Can you triple check that the file your uploading is the one you've re-sorted? If it is then we'll have to dig deeper to figure out why we are getting different results.

**Comment by jsayers-bsc** [comment]
> I tested with the file you sent me and it worked correctly.
> 
> Can you triple check that the file your uploading is the one you've re-sorted? If it is then we'll have to dig deeper to figure out why we are getting different results.

Yes that's the most obvious reason, despite me uploading it a couple times. Same file name, in two different locations :(
